js/chatbot/shared/ChatBot.svelte

```diff
@@ -701,12 +714,6 @@
 		border: 1px solid var(--border-color-accent);
 	}
 
-	/* Lists */
-	.message-wrap :global(ol),
```

Comment:
@freddyaboulton Do you know why we added this?

Reply:
I think this maybe existed in the chatbot before and I copied it over from my custom component when mainlined it

Reply:
LGTM though!